.rubocop.yml

```diff
@@ -151,6 +148,11 @@ Style/Documentation:
     - 'lib/dbus/type.rb'
     - 'lib/dbus/xml.rb'
 
+# Buggy in 1.0, fixed in 1.12
```

Comment:
is there reason to not use 1.12?

Reply:
The reason is that I want to alternate maintenance and development, and RuboCop is firmly in the maintenance category, and updating to 1.0 was enough work already.

So, after some fixes and features I'll be ready to try newer versions :-)